pkgs/top-level/all-packages.nix

```diff
@@ -12800,6 +12800,10 @@ let
     fltk = fltk13;
   };
 
+  zam-plugins = callPackage ../applications/audio/zam-plugins { inherit (xlibs) libX11; };
+  zam-plugins-git = callPackage ../applications/audio/zam-plugins/zam-plugins-git-2015-04-06.nix { inherit (xlibs) libX11; };
```

Comment:
You not need the version in the filename, typically the filename `git.nix` is used for git versions.  Other than that everything looks good.